eng/scripts/Get-PkgVersion.ps1

```diff
@@ -0,0 +1,38 @@
+[CmdletBinding()]
+param (
+
+    [Parameter(Mandatory = $true)]
+    [string] $ServiceDirectory,
+    [Parameter(Mandatory = $true)]
+    [string] $PackageName
+)
+
+. ${PSScriptRoot}/SdkVersion-Common.ps1
+
+$versionFileLocation = Get-VersionHppLocaiton `
+    -ServiceDirectory $ServiceDirectory `
+    -PackageName $PackageName
+
+if (!$versionFileLocation) {
```

Comment:
Falls back to unblock storage artifact generation (docs, changelog, tag metadata)